Directory.Build.props

```diff
@@ -13,6 +13,7 @@
 
     <!-- static local methods are a C# 8.0 feature -->
     <LangVersion>preview</LangVersion>
+    <NullableContextOptions>enable</NullableContextOptions>
```

Comment:
Added this since I was switching between older and newer compilers. Will remove.

Reply:
We have moved to the newest compiler now. This should no longer be required.